azure-pipelines.yml

```diff
@@ -19,9 +19,12 @@ variables:
       value: ''
 
 trigger:
-  - master
-  - release/*
-  - internal/release/6.*
+  batch: true
+  branches:
+    include:
+      - master
+      - release/*
+      - internal/release/6.*
```

Comment:
nit: these dashes could line up with `include:`

Reply:
Since it is subsection I would prefer to keep it indented. It also aligns with the documentation.